atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -90,9 +90,10 @@ def get_next_release_append(self, component, version, base_release):
         # This is brute force, but trying to use getNextRelease() would be fragile
         # magic depending on the exact details of how koji increments the release,
         # and we expect that the number of builds for any one base_release will be small.
+        release = base_release or '0'
```

Comment:
Should it be 1 instead? At least that's what Koji does.

Reply:
Oh, agreed. Fixed